CONFIGURATION.md

```diff
@@ -0,0 +1,90 @@
+
+![¡](docs/plan.png)
+
+#### Adding a AWS account
+
+*On the "Organizations and channels" menu choose "AWS account" and click on the "add AWS account" button.
+
+1. **Name:** Add a name for the account.
+2. **Access Key:** The Access Key for the API user (The user was created in the AWS Configuration section, and the credentials are available in AWS Secrets Manager).
+3. **Secret Access Key:** The Secret Access Key for the API user (The user was created in the AWS Configuration section, and the credentials are available in AWS Secrets Manager).
+4. **Region:** The region in which your AWS Account is deployed.
+5. **MediaConvert Role:** On the AWS IAM service, on the "roles" section there is a search bar, which can be used to get the MediaConvertRole value.
+6. **MediaConvert Endpoint URL** This value is on the "account" section of the Media Convert service in AWS.
+7. **MediaLive Role:** On the AWS IAM service, on the "roles" section there is a search bar, which can be used to get the MediaLiveAccessRole value.
+8. **Account Id:** Your AWS account number.
+
+![¡](docs/aws-account.png)
+
+#### Creating an Organization
+
+*On the "Organizations and channels" menu choose "Organization" and click on the "add organization" button.
+
+1. **Name:** Add a name for your organization. This step will create an s3 bucket with the name of the organization, the name must follow the bucket naming rules: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html .
+2. **Plan:** From the dropdown choose the plan previously created.
+3. **AWS Account:** From the dropdown choose the AWS Account previously created.
+
+![¡](docs/organization.png)
+
+#### Associating an Organization to a user
+
+*On the "Authentication and Authorization" menu choose "Users" and click on the username to edit.
+
+1. **Organization:** Assign the newly created Organization to the superuser to enable authentication via the User web.
+
+![¡](docs/auth.png)
+
+#### Periodic Tasks Configuration
+
+*All kind of tasks can be set to run periodically. This can be done on the "Periodic Tasks" menu.
+
+1. **Clocked:** Here an specific date and hour can be set to run a task.
+2. **Crontabs** Cron configuration, the cron can be configured to run at specific months of the year, specific days of the week, and even specify down to the minute when a particular task should be executed.
+3. **Intervals:** Set intervals to wait before running the same task again, it can be set in days, hours, minutes, seconds, and microseconds.
+4.**Periodic Tasks** Here a Name must be set for the task, and choose a tasl from "Task(registered)" dropdown. Inside the Schedule menu we can choose from the previously created options when we want the task to be executed.
+5. **Solar Events** The task can be executed on a specific zone, this can be set by adding "Latitue" and "Longitude" and selecting one of the solar events from the dropdown menu.
```

Comment:
```suggestion
5. **Solar Events:** The task can be executed in a specific zone, this can be set by adding "Latitude" and "Longitude" and selecting one of the solar events from the dropdown menu.
```

Reply:
we should also change the dropdown word here.

Reply:
added